frontend/apps/filemanager/filemanager.lua

```diff
@@ -48,13 +48,11 @@ end
 local function truncatePath(text)
     local screen_width = Screen:getWidth()
     local face = Font:getFace("xx_smallinfofont")
-    local reversed_text = string.reverse(text)
+    local reversed_text = require("util").utf8Reverse(text)
     local txt_width = RenderText:sizeUtf8Text(0, screen_width, face, reversed_text, nil, false).x
```

Comment:
Just for consistency, I would use `false` insteal of `nil` for this 5th argument (`kerning`). Same 2 lines below in `RenderText:truncateTextByWidth()`.

Also because (if I understand correctly what kerning is) size of text and size of the reverse of text could be different when kerning is involved - so what you do (truncating the reverse) is only fine when no kerning.
(I don't even know if we use kerning at all anywhere).

I would also add a comment:
-- we want to truncate text on the left, so work with the reverse of text (which is fine as we don't use kerning)